registry.json

```diff
@@ -1,4 +1,19 @@
 [ 
+  {
+ "ticker": "ACID",                                                      
+ "poolId": "af6254651bde1e60fb9160d5f321010de5b2fa794e9b1a1163e592e0", 
+ "social": {                                                           
+  "facebook": "https://www.facebook.com/groups/acidpools", 
+  "twitter": "@AcidStakePool",
+   "website": "http://acidpools.com",     
+   "reddit": "",  
+   "instagram": "",
+   "youtube": ""
+  },
+   "telegram": "ACIDpools",                                            
```

Comment:
We need a personal telegram handle. this is to verify voting resaults.

Reply:
Hi leo, I have edited it, my personal telegram handle is _stiiif23_
Thanks